File: app/src/main/java/com/zrq/test/point/demo/activity/Activity2.java
package com.zrq.test.point.demo.activity;

import android.app.Activity;
import android.graphics.Color;
import android.os.Bundle;
import android.widget.TextView;

import androidx.annotation.Nullable;

import com.zrq.test.point.annotation.TestEntryPoint;

/**
 * 描述：
 *
 * @author zhangrq
 * createTime 2020/12/7 16:52
 */
@TestEntryPoint("App-原生-Activity22222222")
public class Activity2 extends Activity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        TextView textView = new TextView(getApplicationContext());
        textView.setText("App-原生-Activity");
        textView.setTextSize(100);
        textView.setTextColor(Color.RED);
        setContentView(textView);
    }
}

File: module-kotlin/src/main/java/com/zrq/test/point/demo/kotlin/activity/Activity2.kt
package com.zrq.test.point.demo.kotlin.activity

import android.app.Activity
import android.graphics.Color
import android.os.Bundle
import android.widget.TextView
import com.zrq.test.point.annotation.TestEntryPoint

/**
 * 描述：
 *
 * @author zhangrq
 * createTime 2020/12/7 16:52
 */
@TestEntryPoint("module-kotlin-原生-Activity")
class Activity2 : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val textView = TextView(applicationContext)
        textView.text = "module-kotlin-原生-Activity"
        textView.textSize = 70f
        textView.setTextColor(Color.RED)
        setContentView(textView)
    }
}